### DISCUSSION TITLE
Data ingestion issue while creating a new program.

### DISCUSSION THREAD

**Discussion by rohandixitdelhi95** [Sunbird-cQube/community#824] [General]
We have been encountering some errors during the data ingestion process while running the following three commands:
1. yarn cli nuke-db
2. yarn cli ingest
3. yarn cli ingest-data
Specifically, we are facing an issue related to incorrect data points when executing the third command. We would greatly appreciate your assistance in resolving this matter.
For your reference, I have attached an error image.
Your support and guidance in rectifying this issue would be highly valued.

![image](https://github.com/Sunbird-cQube/community/assets/140483814/d251c7dc-761f-4bc5-b9cf-083e04755dd0)

**Comment by bommalla** [comment]
@rohandixitdelhi95, Please check the below points.
1. check the data type of the columns
2. check the event data columns and grammar files both correct or not ?
3. check the foreign key relation of the all columns.

**Reply by rohandixitdelhi95** [reply]
Checked and still getting the same error.

**Comment by bommalla** [comment]
Please check the above screenshot "no relevant events for datasets" mean data is incorrect I guess please check the complete data once.
1. please send me the below mentioned  path screenshot with all the data and grammar files
/opt/nifi/nifi-current/Sunbird-cQube-processing-ms/impl/c-qube/ingest/NVSK/dimensions

**Reply by rohandixitdelhi95** [reply]
Send on mail.

**Comment by rohandixitdelhi95** [comment]
Rajesh changed the date format as you told us, still getting the same error.


![MicrosoftTeams-image](https://github.com/Sunbird-cQube/community/assets/140483814/ad5fb865-5a78-4f02-8373-2d9fb82a7ca9)

**Comment by RNarmatha** [comment]
@bommalla - Please check this issue

**Comment by RNarmatha** [comment]
Issue got fixed. Cloumn name for Date and count is changed
